sdk/src/main/java/com/kin/ecosystem/poll/presenter/PollWebViewPresenter.java

```diff
@@ -86,9 +87,12 @@ public void onResponse(OpenOrder response) {
 
 			@Override
 			public void onFailure(KinEcosystemException exception) {
-				eventLogger.send(EarnOrderCreationFailed.create(exception.getCause().getMessage(), offerID));
-				if (view != null) {
-					showToast(exception.getMessage());
+				showToast("Oops something went wrong...");
```

Comment:
1. string should be in strings.xml 
2. Why not displaying the error message in the toast (it will help us debug the problem)